### PR TITLE
Add support for containers

### PR DESCRIPTION
**Pull request by l0kod**
This will make easier to use specific tools (e.g. linters) and test against different distros.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,18 @@ built with the kernels/make-uml.sh script.
 
 uml-run.sh can be used to launch an UML kernel with an init test script.
 
+## docker-run
+
+Build a container to build the kernel, samples, tests and check everything for Landlock.
+
+Required installed and configured softwares: docker and optionally docker-buildx.
```
Comment (gnoack) [inline README.md]: typo: s/softwares/software/

**Comment by l0kod** [comment]
I'm holding this PR for now because tests don't pass with the latest kernel. It's related to landlock-lsm/linux@0710a1a73fb45033ebb06073e374ab7d44a05f15 (and landlock-lsm/linux@41cca0542d7c79b6be3920a917f643c075ad1561), because of private mount points.

**Comment by l0kod** [comment]
See https://lore.kernel.org/all/20240426172252.1862930-1-mic@digikod.net/

**Comment by l0kod** [comment]
> I'm holding this PR for now because tests don't pass with the latest kernel. It's related to [landlock-lsm/linux@0710a1a](https://github.com/landlock-lsm/linux/commit/0710a1a73fb45033ebb06073e374ab7d44a05f15) (and [landlock-lsm/linux@41cca05](https://github.com/landlock-lsm/linux/commit/41cca0542d7c79b6be3920a917f643c075ad1561)), because of private mount points.

Fixed in Linux 6.9 with landlock-lsm/linux@af300a3959290b005f27ab5858bfebcb4840cd66

**Pull request merged**